Sources/Benchmark/OperatingSystemStats/OperatingSystemStatsProducer+Darwin.swift

```diff
@@ -65,6 +65,23 @@
                 }
                 return procTaskInfo
             }
+
+            struct IOStats {
+                var bytesRead, bytesWritten: UInt64
+            }
+
+            private func getIOStats() -> IOStats {
+                var rinfo = rusage_info_v2()
```

Comment:
Maybe use `rusage_info_v6` instead, then we can look at adding support for instruction and cycle count easily as a future direction?

I tried also with  [ri_logical_writes](https://developer.apple.com/documentation/kernel/rusage_info_v4/2870525-ri_logical_writes) to get either support for either `writeSyscalls` or `writeBytesLogical` but the values returned doesn't make much sense (and couldn't find definition by searching xnu)

Reply:
Just used the lowest version I needed.